src/python/pants/backend/project_info/filedeps.py

```diff
@@ -103,12 +103,12 @@ async def file_deps(
             itertools.chain.from_iterable(tgt.get(Sources).filespec["includes"] for tgt in targets)
         )
     else:
-        all_hydrated_sources = await MultiGet(
-            Get(HydratedSources, HydrateSourcesRequest(tgt.get(Sources))) for tgt in targets
+        all_sources_paths = await MultiGet(
+            Get(SourcesPaths, SourcesPathsRequest(tgt.get(Sources))) for tgt in targets
```

Comment:
Please feel free to bikeshed these names. I think it's confusing how many types we have related to sources:

* `Sources`, which is the field. Should have probably been `SourcesField` in retrospect.
* `HydratedSources`/`HydrateSourcesRequest`
* `SourcesSnapshot`, i.e. snapshot of all the specs. This should maybe be renamed to `SpecsSnapshot`?
* `SourcesPaths`/`SourcesPathsRequest`
* `SourceRoot`

Also, I made a typo between `SourcePaths` and `SourcesPaths` a few times, which is probably a smell.

Reply:
> This should maybe be renamed to SpecsSnapshot?

https://github.com/pantsbuild/pants/pull/11114